packages/shared/src/components/checklist/EditSquadStep.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import { ChecklistStepPropsWithSquad } from '../../lib/checklist';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { ChecklistStep } from './ChecklistStep';
+import { EditIcon } from '../icons';
+
+export const EditSquadStep = ({
+  squad,
+  ...props
+}: ChecklistStepPropsWithSquad): ReactElement => {
+  return (
+    <ChecklistStep {...props}>
+      <div className="flex">
+        <Button
+          tag="a"
+          variant={ButtonVariant.Primary}
+          icon={<EditIcon />}
+          href={`/squads/${squad.handle}/edit`}
```

Comment:
Is there anyway this could show in extension?
Let's maybe just put `webappUrl` in front to be safe?

Reply:
This is never shown from the extension, but yeah we can add that.